lighthouse-core/gather/driver.js

```diff
@@ -291,11 +291,25 @@ class Driver {
         err.message += ` Method: ${method}`;
         reject(err);
       }), timeout);
+
+      const status = {
+        msg: `Devtools Protocol Command ${method}`,
+        id: `lh:driver:sendCommand:${method}`,
+      };
+      if (log.isVerbose()) {
+        log.time(status, 'verbose');
+      }
       try {
         const result = await this._connection.sendCommand(method, ...params);
+        if (log.isVerbose()) {
```

Comment:
can we stick this in a `finally` instead?

Reply:
It wouldn't be the same functionality - in `finally`, the resolve/reject cb would happen first.

Reply:
Oh I'm sensing a learning opportunity for me! :D 

What situations raise an error or impact the timings in any meaningful way? Until today I thought the callbacks would just be run in a microtask afterwards and resolve/reject is basically a super quick bitflip with a bit of state saved

Reply:
Ok, clearly this just registers a value/promise to resolve to, doesn't actually execute it or anything. Slight confusion :)